src/views/SystemRepositoryTable/index.tsx

```diff
@@ -0,0 +1,180 @@
+import classnames from 'classnames';
+
+import Footer from 'components/Footer';
+import Header from 'components/Header';
+import MainContent from 'components/MainContent';
+import PageWrapper from 'components/PageWrapper';
+import { NavLink, SecondaryNav } from 'components/shared/SecondaryNav';
+import SystemHealthIcon from 'components/SystemHealthIcon';
+import { IconStatus, sortByStatus } from 'types/iconStatus';
+import { mockSystemInfo, SystemInfo } from 'views/Sandbox/mockSystemData'; // TODO - replace mockSystemInfo with dynamic data fetched from backend
+
+// TODO - if we want to keep this text past the prototype state, it needs to use translation
+const mapStatusToDescription = (status: IconStatus): string => {
+  switch (status) {
+    case 'success':
+      return 'Fully operational';
+    case 'warning':
+      return 'Degraded functionality';
+    case 'fail':
+    default:
+      return 'Inoperative';
+  }
+};
+
+export const SystemRepositoryTable = () => {
+  const { t } = useTranslation('systemProfile');
+
+  const sortRowsByStatus = useMemo(
+    () => (rowA: Row<SystemInfo>, rowB: Row<SystemInfo>) =>
+      sortByStatus(
+        rowA.original.productionStatus,
+        rowB.original.productionStatus
+      ),
+    []
+  );
+
+  const columns = useMemo<Column<SystemInfo>[]>(() => {
+    return [
+      {
+        Header: t<string>('systemTable.header.systemName'),
+        accessor: (systemInfo: SystemInfo) => `System ${systemInfo.name}`,
+        id: 'systemName',
+        Cell: ({ row }: { row: Row<SystemInfo> }) => (
+          <UswdsLink asCustom={Link} to={`/sandbox/${row.original.id}`}>
+            {row.original.name}
+          </UswdsLink>
+        )
+      },
+      {
+        Header: t<string>('systemTable.header.systemOwner'),
+        accessor: (systemInfo: SystemInfo) =>
+          `${systemInfo.ownerName}, ${systemInfo.ownerOffice}`,
+        id: 'systemOwner'
+      },
+      {
+        Header: t<string>('systemTable.header.productionStatus'),
+        accessor: 'productionStatus',
+        Cell: ({ row }: { row: Row<SystemInfo> }) => (
+          <SystemHealthIcon
+            status={row.original.productionStatus}
+            size="medium"
+            label={mapStatusToDescription(row.original.productionStatus)}
+          />
+        ),
+        sortType: sortRowsByStatus
+      }
+    ];
+  }, [t, sortRowsByStatus]); // TODO when system data is dynamically fetched, this dependency list may need to be changed
+
+  const {
+    getTableProps,
+    getTableBodyProps,
+    headerGroups,
+    rows,
+    prepareRow
+  } = useTable(
+    {
+      columns,
+      data: useMemo(() => mockSystemInfo, []),
+      initialState: {
+        sortBy: useMemo(() => [{ id: 'systemName', desc: false }], [])
+      }
+    },
+    useSortBy
+  );
+
+  // TODO - copied from src/components/RequestRepository/index.tsx. is this something we should generalize & reuse?
```

Comment:
This could probably be moved to a helper, but I don't know if it's critical that it be done as part of this ticket. I'm tempted to just move this through because it'll be the base for the other UI tickets in the pipeline, and it's not a critical issue.

Reply:
Sounds good to me.